controllers/application_controller.go

```diff
@@ -149,6 +187,7 @@ func (r *ApplicationReconciler) reconcileApplication(ctx context.Context, argocd
 		logger.Error(err, "failed to sync application status")
 		return ctrl.Result{}, err
 	}
+	logger.Info("Application successfully reconciled")
```

Comment:
Clarifying the application name is better; or it would be good to be a debug level message.